### DISCUSSION TITLE
中央的日語課程

### DISCUSSION THREAD

**Discussion by seastar0687** [1101-GS4719-C/Python-Community-GS4719-C#3708] [General]
中央的日語課程分成1A，1B，2A，2B，3A
每個都是一學期的課程，且要修完前一階段的才能修下一階段
我自己目前修到2A，我的感覺是覺得這個課程安排對於有心想學日語的人士非常慢的
1A，1B花兩個學期才學完大家的日本語第一本，實際上可以學得更快很多
我覺得去修這門課程大部分還是拿學分的動機比較大，如果想學的話還是而外去補習或是請家教會比較好

大家怎麼看呢

**Comment by SendaiHatsuki** [comment]
個人覺得設計的真的不好，首先是擋修的部分，我本身是有點基礎，但還是要從最基礎的50音開始學，真的很讓費時間。我覺得還可以新增不同的班，像是口說班專門教口說，檢定班專門教日檢的。

**Comment by XiangLuen** [comment]
我認為學校日文班不能這樣設計，因為有部分學生可能高中就有修過相關的日文課本，已經具備相關的日文能力了，但還是要從1A開始上。如果可以透過提交日文檢定或是在選課時分成日文初中高級班，並告知每個班會學習哪種等級像是N5~N4為初級班，N3為中級，N2~N1為高級，然後就讓學生自由選擇自己程度的課程，這樣會對學生的受教以及教師教學比較方便。
